spacecmd/src/lib/api.py

```diff
@@ -30,7 +30,6 @@
     from xmlrpc import client as xmlrpclib
 except ImportError:
     import xmlrpclib
-from utils import CustomJsonEncoder
```

Comment:
Now that I am reviewing again, if this is breaking compatibility with python3, it needs to be fixed so it works with python2 as well, as spacecmd is used at the client tools as well :-)

Why are we removing this import? Is it because it's not used it anywhere at the code?

Reply:
@juliogonzalez The line right after this one is:

    from spacecmd.utils import *

which basically import `CustomJsonEncoder` and more and is compatible with both py2 and py3: py3 doesn't like relative imports.

Reply:
@cbosdo, if we keep compatibility with python2, then this is ready to merge.

Merging and submitting as part of Alpha2